### DISCUSSION TITLE
Homework 5 Question

### DISCUSSION THREAD

**Discussion by RyanW-UConn** [cooperrc/advanced-dynamics#42] [General]
I see in the diagram it shows $z_2 = 0.5 x_{22} $. Is this supposed to say $z_2 = 0.5x_2^2$? If the track is a parabola then the position of $z_2$ should be proportional to the square of $x_2$ I would think.

![image](https://github.com/cooperrc/advanced-dynamics/assets/157430741/99723b64-a15d-4b59-8104-84aa7e8936f9)

Thanks!

**Comment by travismalave** [comment]
Yea that's correct - the diagram I see in homework 5 shows $z_2=0.5*(x_2)^2$. Not sure what happened in that figure.
